linux/ibus-keyman/build.sh

```diff
@@ -19,10 +19,8 @@ builder_describe \
   "test" \
   "install                   install artifacts" \
   "uninstall                 uninstall artifacts" \
-  "--debug,-d                Debug build"
-# We can't yet depend on core until it moved to the new build.sh syntax
-# (currently it doesn't know some parameters that we're passing)
-#  "@/core configure build"
+  "--debug,-d                Debug build" \
+  "@/core :arch"
```

Comment:
This won't work the way you are expecting -- we can't specify dependency targets yet, so this would attempt to build core if you specified `:arch` on ibus-keyman, which wouldn't work. I will think this through...